app/src/js/components/Providers/provider.js

```diff
@@ -10,14 +10,17 @@ import Loading from '../LoadingIndicator/loading-indicator';
 import LogViewer from '../Logs/viewer';
 import ErrorReport from '../Errors/report';
 import Metadata from '../Table/Metadata';
-import {
-  getPersistentQueryParams,
-  historyPushWithQueryParams,
-} from '../../utils/url-helper';
 import { metaAccessors } from '../../utils/table-config/providers';
+import { withUrlHelper } from '../../withUrlHelper';
+
+const ProviderOverview = ({ urlHelper }) => {
+  const dispatch = useDispatch();
+  const { providerId } = useParams();
+  const { location, getPersistentQueryParams, historyPushWithQueryParams } = urlHelper;
+
+  const providers = useSelector((state) => state.providers);
+  // const logs = useSelector((state) => state.logs); -- logs was declared in previous class component but not used
```

Comment:
Reminder to remove comment

Reply:
I put that comment there because I'm not sure if the logs state should be removed it was there but undefined but there's a reference to the logviewer and since I was just working on routes I didn't want to remove that it has to be refactored later.

Reply:
Can we add a note to that effect in that case? e.g. why this code is commented out and what the plan is, briefly?

Reply:
What else should I add to the note [`logs was declared in previous class component but not used`] that's there?